src/feedrattler/convert.py

```diff
@@ -1,24 +1,26 @@
-from typing import Optional
-import os
-import tempfile
+import contextlib
 import logging
+import os
 import pathlib
-import contextlib
 import shutil
+import tempfile
 import time
-
-from github import UnknownObjectException
-from git import Repo
+from typing import Optional
 
 from conda_recipe_manager.commands.convert import convert_file
 from conda_recipe_manager.commands.utils.types import ExitCode
 from conda_smithy import configure_feedstock
+from git import Repo
+from github import UnknownObjectException
 
-from .utils import initialize_yaml
-from .utils import update_python_min_in_recipe
-from .utils import update_python_version_in_tests
-from .utils import remove_empty_script_test
-from .utils import CloneType
+from .utils import (
```

Comment:
Imports were sorted by ruff's "organize imports". I can revert that change if you want.

Reply:
All good.

Actually, if you want to configure ruff to sort the import in the pyproject.toml file, you are welcome :-)